platforms/enterprise/enterprise/src/integTest/groovy/org/gradle/internal/enterprise/GradleEnterprisePluginBackgroundJobExecutorsIntegrationTest.groovy

```diff
@@ -33,7 +33,7 @@ class GradleEnterprisePluginBackgroundJobExecutorsIntegrationTest extends Abstra
     @Rule
     BlockingHttpServer httpServer = new BlockingHttpServer()
 
-    def plugin = new GradleEnterprisePluginCheckInFixture(testDirectory, mavenRepo, createExecuter())
+    def plugin = new DevelocityPluginCheckInFixture(testDirectory, mavenRepo, createExecuter())
```

Comment:
💬 My approach to testing was to exchange GEFixture with DevelocityFixture. This way most of the tests now run against the Develocity plugin, and only a couple against the GE plugin.